api/services/reviewService.js

```diff
@@ -0,0 +1,52 @@
+const { reviewDao } = require("../models");
+const { commentValidate } = require("../utils/validate-check");
+const { reviewValidate } = require("../utils/validate-check");
+const { userMovieValidate } = require("../utils/validate-check");
+const jwt = require("jsonwebtoken");
```

Comment:
```suggestion
const jwt = require("jsonwebtoken");

const { reviewDao } = require("../models");
const { 
  commentValidate,
  reviewValidate,
  userMovieValidate
} = require("../utils/validate-check");
```
1. 프로그래밍의 범용적인 컨벤션에 따라 built-in 모듈 -> 외부 모듈 -> custom 모듈 순으로 import 하는 코드가 쓰일 수 있도록 해주세요.
2. 같은 파일에서 모듈을 import하는 것이라면 각각 쓰는 것이 아닌, 한줄로 쓰이는게 유지보수에 편리하겠습니다.